tests/test_cosine_similarity.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+import nlu
+
+
+def test_cosine_similarity():
+    question_segment = ['我', '要', '查', '成績']
```

Comment:
For the best reading feeling with these tests, I say:
```py
def similarity(a, b):
  nlu._consine_similarity(jieba.cut(a), jieba.cut(b))

def test_cosine_similarity():
  question_segment = '我要查成績'
  # same as previous format
  _ = similarity(question_segment, user_input_segment)
  # same as original code
```

Reply:
sure